src/State.h

```diff
@@ -144,8 +145,10 @@ class State
 
 	inline ~State()
 	{
+#ifdef TRACY_ENABLE
```

Comment:
When no TRACY_ENABLE, `TracyD3D11Destroy` is interpreted as empty strings, so no need

Reply:
Matter of fact I don't think tracy is designed to require manual TRACY_ENABLE checks anywhere

Reply:
Caused an warning without this
<img width="1069" alt="image" src="https://github.com/user-attachments/assets/38cfaa3f-980c-4bbf-8d36-03b37a33902f">

Reply:
uhhhhh I think we should turn this warning off